misk-launchdarkly-core/src/main/kotlin/misk/feature/launchdarkly/LaunchDarklyFeatureFlags.kt

```diff
@@ -111,6 +111,9 @@ class LaunchDarklyFeatureFlags @Inject constructor(
     if (attributes.number != null) {
       attributes.number!!.forEach { (k, v) -> builder.privateCustom(k, v) }
     }
+    if (attributes.anonymous) {
+      builder.anonymous(true)
```

Comment:
this makes it so that launchdarkly doesn't create a "User" on their side

Reply:
I think this is worth noting in a comment